centreon/www/include/common/listing/listing.css

```
@@ -1928,13 +1928,13 @@
 }
 .cld-diff-table .cld-fbefore {
     background: rgba(255, 74, 74, 0.05);
-    color: #c0392b;
+    color: var(--color-danger-dark, #D60101);
     font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)